src/ec/packages/ec-component-site-header-standardised/site-header-standardised.html.twig

```diff
@@ -0,0 +1,316 @@
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<header class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  <div class="ecl-site-header-standardised__container ecl-container">
+    <div class="ecl-site-header-standardised__top">
+      <a
+        class="ecl-link ecl-link--standalone ecl-site-header-standardised__logo-link"
+        href="{{ _logo.href }}"
+        aria-label="{{ _logo.title }}"
+      >
+        <img
+          alt="{{ _logo.alt }}"
+          title="{{ _logo.title }}"
+          class="ecl-site-header-standardised__logo-image"
+          src="{{ _logo.src }}"
+        />
+      </a>
+      <div class="ecl-site-header-standardised__action">
+      {% if _login_toggle is not empty and _login_box is not empty %}
+        <div class="ecl-site-header-standardised__login-container">
+        {% if _logged %}
+          <a
+            class="ecl-link ecl-link--standalone ecl-site-header-standardised__login-toggle"
+            href="{{ _login_box.href_logged }}"
+            aria-controls={{ _login_box.id }}
+            aria-expanded="false"
+          >
+            {% include '@ecl-twig/ec-component-icon/icon.html.twig' with {
+              icon: {
+                path: _icon_file_path,
+                type: 'general',
+                name: 'logged-in',
+                size: 's'
+              }
+            } only %}
+             {{- login_toggle.label_logged -}}
+          </a>
+          <div
+            id="{{ _login_box.id }}"
+            class="ecl-site-header-standardised__login-box"
+            data-ecl-login-box
+          >
+            <p class="ecl-site-header-standardised__login-description">
+              {{- _login_box.description|raw -}}
+            </p>
+            <hr className="ecl-site-header-standardised__login-separator" />
+            {% include '@ecl-twig/ec-component-link/link.html.twig' with {
+              link: {
+                label: _login_box.label,
+                href: _login_box.href,
+                variant: "standalone"
+              }
+            } only %}
+          </div>
+        {% else %}
+          <a
+            class="ecl-link ecl-link--standalone ecl-site-header-standardised__login-toggle"
+            href="{{ _login_box.href_not_logged }}"
+          >
+            {% include '@ecl-twig/ec-component-icon/icon.html.twig' with {
+              icon: {
+                path: _icon_file_path,
+                type: 'general',
+                name: 'log-in',
+                size: 's'
+              }
+            } only %}
+            {{- login_toggle.label_not_logged -}}
+          </a>
+        {% endif %}
+        </div>
+      {% endif %}
+      {% if _language_selector is not empty %}
+        <a
+          class="ecl-link ecl-link--standalone ecl-site-header-standardised__language-selector"
+          href="{{ _language_selector.href }}"
+          data-ecl-language-selector
```

Comment:
data-ecl-language-selector="true" ?

Reply:
same with data-ecl-language-list-overlay="true" coming from language-list component.

Reply:
yeah, but what is surprising you, the fact that you see in the inspector the "true" value? If you check in the html tab these attributes are only set by their name. Was this the concern you had?

Reply:
It was more regarding this issue https://github.com/ec-europa/ecl-twig/issues/165, if now we need to always set the data-attribute values even if it's true? But anyway let merge this and see this more globally if needed.